CMakeLists.txt

```diff
@@ -26,6 +26,8 @@ option(UMF_BUILD_BENCHMARKS "Build UMF benchmarks" OFF)
 option(UMF_ENABLE_POOL_TRACKING "Build UMF with pool tracking" ON)
 option(UMF_DEVELOPER_MODE "Enable developer checks, treats warnings as errors" OFF)
 option(UMF_FORMAT_CODE_STYLE "Format UMF code with clang-format" OFF)
+option(USE_ASAN "Enable AddressSanitizer checks" OFF)
```

Comment:
we should document these options in README

Reply:
Done.